app/controllers/applications_controller.rb

```diff
@@ -16,6 +16,15 @@ def create
     end
   end
 
+  def update
+    @application = Application.find(params[:id])
+    if params[:pet]
```

Comment:
This was a much cleaner solution than my application_pets#create idea! Great job!